azure_arc_servers_jumpstart/docs/arc_policies_mma.md

```diff
@@ -0,0 +1,113 @@
+
+You can use the Azure Portal, Azure CLI, an ARM template or PowerShell script to assign policies to Azure Subscriptions or Resource Groups. In this guide, you will use an ARM template to assign built-in policies. 
+
+**Note: This guide assumes you already deployed VMs or servers that are running on-premises or other clouds and you have connected them to Azure Arc.**
+
+**If you haven't, this repository offers you a way to do so in an automated fashion:**
+- **[GCP Ubuntu VM](gcp_terraform_ubuntu.md) / [GCP Windows VM](gcp_terraform_windows.md)**
+- **[AWS Ubuntu VM](aws_terraform_ubuntu.md)**
+- **[VMware Ubuntu VM](vmware_terraform_ubuntu.md) / [VMware Windows Server VM](vmware_terraform_winsrv.md)**
+- **[Local Ubuntu VM](local_vagrant_ubuntu.md) / [Local Windows VM](local_vagrant_windows.md)**
+
+# Prerequisites
+
+* Clone this repo.
+
+* As mentioned, this guide starts at the point where you already deployed and connected VMs or servers to Azure Arc.In the screenshots below we can see a GCP server has been connected with Azure Arc and is visible as a resource in Azure.
+
+    ![](../img/vm_policies/01.png)
+
+    ![](../img/vm_policies/02.png)
+
+  
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 2.7** or later. Use ```az --version``` to check your current installed version.
+
+* Create Azure Service Principal (SP)   
+
+    To connect a VM or bare-metal server to Azure Arc, Azure Service Principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+  ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+    For example:
+    ```az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor```
+    Output should look like this:
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+    
+**Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).
+
+* You will also need to have a Log Analytics Workspace deployed. You can automate the deployment by editing the ARM template [parameters file](../policies/arm/log_analytics-template.parameters.json) and provide a name and location for your workspace. 
+
+    ![](../img/vm_policies/03.png)
+
+Then start the deployment with the command:
+
+    ```bash
+    az deployment group create --resource-group <resource-group-name> --template-file <path-to-template> --parameters <path-to-parametersfile>
+    ```
```

Comment:
```suggestion
* You will also need to have a Log Analytics Workspace deployed. You can automate the deployment by editing the ARM template [parameters file](../policies/arm/log_analytics-template.parameters.json), provide a name and location for your workspace. To start the deployment, use the below command:

    ```bash
    az deployment group create --resource-group <resource-group-name> --template-file <path-to-template> --parameters <path-to-parametersfile>
    ```

    ![](../img/vm_policies/03.png)

```